Microsoft.Azure.CosmosRepository/test/Providers/DefaultCosmosPartitionKeyPathProviderTests.cs

```diff
@@ -28,6 +27,6 @@ public class PickleChipsItem : Item
         [JsonProperty(PropertyName = "pickles")]
         public string PartitionBits { get; set; } = "Hey, where's the chips?!";
 
-        protected override string GetPartitionKeyValue() => PartitionBits;
+        protected override string GetPartitionKeyValue() => this.PartitionBits;
```

Comment:
```suggestion
        protected override string GetPartitionKeyValue() => PartitionBits;
```

The `this` qualifier is redundant.

Reply:
The other reason to use this in these situations is to distinguish between a static and an instance variable. It just prevents you from accidentally doing silly things. I'll go flip the config around a bit and reformat things.